CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ All notable changes to this project will be documented in this file.
 * ### ALL
   * #### Added
   * #### Changed
+    * Changed abort() to be a public function.
```

Comment:
Clients don't care about public/private. That's an implementation detail and we typically only list things here that affect clients.
So you didn't change abort. You added it.